confgenerator/testdata/valid/windows/metrics-receiver_mssql_v2_override/input.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics:
```

Comment:
I don't see this override input.yaml file is any different from `confgenerator/testdata/valid/windows/metrics-receiver_mssql_v2_duplicate/input.yaml`. Am i missing something? 🤔

Reply:
@sophieyfang one of these shares the same receiver name as the default mssql that is run (so the version 2 will override the version 1). The other config has different names so that version 1 and 2 run in tandem.

Reply:
Sync'd with sophie offline. We can resolve this.